### ISSUE TITLE
Crash on 1.13.2 Forge 25.0.59

### ISSUE THREAD

**Issue by RedBullSlurpie** [SquidDev-CC/CC-Tweaked#131]
## Useful information to include:
Forge version is 25.0.59
 - Minecraft version 1.13.2
 - CC: Tweaked version cc-tweaked-1.13.2-1.81.1
 - Detailed reproduction steps: sometimes I can spot a bug pretty easily, but often it's much more obscure. The more information I have to help reproduce it, the quicker it'll get fixed.
Launch the game, and the game crashes.

Pastebin for crash log- https://pastebin.com/jrhb3JE3

**Comment by SquidDev** [comment]
Weird - I've not had any issues with 25.0.59 or a dev environment - does this happen every time?

**Comment by RedBullSlurpie** [comment]
Seems to yes. I do have other mods installed, would you like a list of them ?

**Comment by RedBullSlurpie** [comment]
![capture](https://user-images.githubusercontent.com/9391592/53459152-c15fb300-3a06-11e9-972d-6125aa148580.PNG)

**Comment by SquidDev** [comment]
Hrmrrm. I'm fairly sure this is not a CC:T bug - I'm registering items correctly - so I assume it's an issue with Forge's new threaded mod loader. Do you think you could upload your full `debug.log`, so I can see what's being registered?

**Comment by RedBullSlurpie** [comment]
I don't seem to have a debug file in the folder. But I actually figured out what mod is causing it. It is ImprovableSkills-1.13.2-3.70r that is causing your mod to come up as the crash culprit. But when I remove ImprovableSkills-1.13.2-3.70r, your mod works fine. I submitted a bug report to him earlier although that was before I knew his mod was causing yours to throw up that error in the crash report. Hopefully when the dev of that other mod fixes the issue on there side, itll be resolved. :) Sorry for wasting your time!
